pype_schema/tag.py

```diff
@@ -736,16 +476,13 @@ def calculate_values(self, data, tag_to_var_map={}):
         list, array, or Series
             numpy array of combined dataset
         """
-        if self.unary_operations is not None:
-            data = self.process_unary_ops(data, tag_to_var_map=tag_to_var_map)
-
-        if self.binary_operations is not None:
-            data = self.process_binary_ops(data, tag_to_var_map=tag_to_var_map)
+        if self.operations is not None and self.operations:
+            data = self.process_ops(data, tag_to_var_map=tag_to_var_map)
         elif isinstance(data, (dict, DataFrame)):
-            # if no binary ops, get appropriate column from unary ops and rename
+            # if ops, get appropriate column and rename
```

Comment:
are these if/else statements still necessary? They were only for edge cases where there were unary operations but not binary operations (and that no longer applies afaik)